README.md

```diff
@@ -285,16 +286,18 @@ mkdir -p /matrix-appservice-slack
 #  database:
 #    userStorePath: "/data/user-store.db"
 #    roomStorePath: "/data/room-store.db"
+#  dbdir: "/data"
 nano /matrix-appservice-slack/config.yaml
 
 # Copy the registration file to the volume
 cp slack-registration.yaml /matrix-appservice-slack/slack-registration.yaml
 
-# Optional: Build the container yourself (requires a git clone, and to be in the root of the project)
-docker build -t halfshot/matrix-appservice-slack .
+# Recommended: Build the container yourself (requires a git clone, and to be in the root of the project)
+docker build -t cadair/matrix-appservice-slack.
```

Comment:
Do we want the `cadair/` here?

(I honestly don't know!)

Reply:
If the official one exists (even if it isn't finished yet), I think that would be best, for the documentation to stay modern longer, but I'm not sure who will own the official one. Do you know?

Reply:
Well there is a dockerfile in the repo, which hopefully will become `matrixdotorg/matrix-appservice-slack` at some point. For these instructions I would just not have a "user" prefix at all?
```
docker build -t matrix-appservice-slack .
docker run [...]  matrix-appservice-slack
```
??